integration_tests/integration/documents.spec.ts

```diff
@@ -123,25 +125,37 @@ context('Documents', () => {
   })
 
   it('should show which column the table is currently sorted by', () => {
-    const headings = ['Name', 'Type', 'Event', 'Date&nbsp;created']
-    headings.forEach(heading => {
-      it(`should set headings correctly when sorting by ${heading}`, () => {
-        cy.get('table').within(() => cy.contains('button', heading).click())
-
-        // check the clicked heading is sorted and all others are not
-        cy.get('thead')
-          .find('th')
-          .each($el => {
-            const sort = $el.text() === heading ? 'ascending' : 'none'
-            cy.wrap($el).should('have.attr', { 'aria-sort': sort })
-          })
-
-        // clicking again sorts in the other direction
-        cy.get('table').within(() => cy.contains('button', heading).click())
-
-        cy.get('table').within(() => cy.contains('button', heading).should('have.attr', { 'aria-sort': 'descending' }))
-      })
-    })
+    const sortExpectations = [
+      {
+        columnHeaderName: 'Name',
+        orderedData: [
+          // todo: someone could look into this? Problem outlined....
```

Comment:
Can we get the upload date and use that instead 🤔

Reply:
this relates to the name column though. There is a sortable `Date created` column already

Reply:
happy to discuss and figure out this columns sort though! But probs out of scope here I was just adding in coverage for this pre-existing sort and found this column should prob not be tested as the sort is broken (which is not a side effect of this PR's changes - has been broke for a while)

Reply:
let me know if you want to discuss further :)

Reply:
Ok then looks fine then